src/modules/GroupChannelList/components/ChannelListHeader/index.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+
+import { LocalizationContext } from '../../../../lib/LocalizationContext';
+import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import Label, { LabelTypography, LabelColors } from '../../../../ui/Label';
+
+import './index.scss';
+import Avatar from '../../../../ui/Avatar';
+
+interface ChannelListHeaderInterface {
+  renderHeader?: (props: void) => React.ReactElement;
+  renderTitle?: () => React.ReactElement;
+  renderIconButton?: (props: void) => React.ReactElement;
+  onEdit?: (props: void) => void;
+  allowProfileEdit?: boolean;
+}
+
+const ChannelListHeader: React.FC<ChannelListHeaderInterface> = ({
+  renderHeader,
+  renderTitle,
+  renderIconButton,
+  onEdit,
+  allowProfileEdit,
+}: ChannelListHeaderInterface) => {
+  const { stores, config } = useSendbirdStateContext();
+  const { user } = stores.userStore;
+  const { logger } = config;
+
+  const { stringSet } = useContext(LocalizationContext);
+
+  if (renderHeader) {
+    logger?.warning('Recomend to use "renderTitle" instead of "renderHeader". It will be deprecated.');
+  }
+  // renderTitle should have higher priority
+  const titleRenderer = renderHeader || renderTitle;
+
+  return (
+    <div
+      className={[
+        'sendbird-channel-header',
+        allowProfileEdit ? 'sendbird-channel-header--allow-edit' : '',
+      ].join(' ')}
+    >
+      {
+        titleRenderer?.() || (
```

Comment:
At the current moment, it's not particularly crucial, but, it seems necessary to discuss whether this rendering approach is suitable for the DX.
In React, typically, if a user does not want to render the component, they can explicitly specify null or undefined as the return value.
However, when using this approach, `Fragment` appears to be the only viable option.

```tsx
renderHeader={() => null} // not working
renderHeader={() => <React.Fragment />} // working
```

Reply:
That's a good idea, we can do that!